Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent {
+        label 'linux-large'
+        //TODO 6146 windows builds ?using label 'proxmox-windows'?
```

Comment:
We might need to fallback to the 'node' scripted syntax. I will see how possible it is to manage agents in the declarative pipelines format.